src/examples.rs

```diff
@@ -97,7 +97,7 @@ impl Default for Examples {
                 description: r#"A Zig fibonacci example."#.to_string()
             },
             Example {
-                disabled: Some(true),
+                disabled: None,
```

Comment:
1. This should just be a boolean (true/false)
2. What is the motivation for this? What has changed? It's still the exact same workload which was causing the issue, right?

Reply:
1. https://github.com/profianinc/benefice/issues/181
2. I tried running it with enarx 0.6.4 and benefice locally and it seems to work. Paul and I wanted to test it in the staging env to see if the demo works.